<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFriendWin</class>
 <widget class="QWidget" name="AddFriendWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查找</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(0, 201, 167, 128);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">border-radius:1px;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="findUser">
      <attribute name="title">
       <string>找人</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>70</height>
          </size>
         </property>
         <widget class="QLineEdit" name="lineEdit">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>30</y>
            <width>301</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius:10px;
background-color: rgba(255, 255, 255, 78);</string>
          </property>
          <property name="placeholderText">
           <string>          请输入ID号</string>
          </property>
         </widget>
         <widget class="QPushButton" name="searchBut">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>20</y>
            <width>91</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border-radius:15px;</string>
          </property>
          <property name="text">
           <string>search</string>
          </property>
         </widget>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>30</y>
            <width>41</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/res/pic/搜索.png);
background-color: rgba(255, 255, 255, 0);</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="resultBut" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 155);
border-radius:10px;</string>
         </property>
         <widget class="QLabel" name="resultLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>91</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>URW Bookman</family>
            <pointsize>20</pointsize>
            <italic>true</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="text">
           <string>reslut:</string>
          </property>
         </widget>
         <widget class="QLabel" name="nameLabel">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>90</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Noto Sans Adlam</family>
            <pointsize>14</pointsize>
            <italic>true</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="text">
           <string>name:</string>
          </property>
         </widget>
         <widget class="QLabel" name="idLabel">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>90</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Noto Sans Adlam</family>
            <pointsize>14</pointsize>
            <italic>true</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="text">
           <string>ID:</string>
          </property>
         </widget>
         <widget class="QLabel" name="headLabel">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>170</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Noto Sans Adlam</family>
            <pointsize>14</pointsize>
            <italic>true</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="text">
           <string>head:</string>
          </property>
         </widget>
         <widget class="QPushButton" name="addBtn">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>210</y>
            <width>91</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border-radius:15px;</string>
          </property>
          <property name="text">
           <string>add</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="findGroup">
      <attribute name="title">
       <string>找群</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
